colorbleed/plugins/global/load/copy_file.py

```diff
@@ -32,4 +32,4 @@ def copy_file_to_clipboard(path):
 
         # Set to Clipboard
         clipboard = app.clipboard()
-        clipboard.setMimeData(mime)
+        clipboard.setMimeData(mime)
```

Comment:
Blegh, is this pycharm doing that? :) Shouldn't it actually force a new line being there?